config/manager/kustomization.yaml

```diff
@@ -9,3 +9,9 @@ configMapGenerator:
 - files:
   - controller_manager_config.yaml
   name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: weaveworks/profiles-controller
+  newName: localhost:5000/profiles-controller
+  newTag: latest
```

Comment:
is this that thing we said we would remember to not commit?

Reply:
Grrrr. We need to ignore this then force push it.